Cares/index.md

```diff
@@ -75,6 +75,15 @@ CARES committee members may also attend other workshops and conferences sponsore
     <th><b>Conference/Meeting</b></th>
     <th><b>CARES member attendees</b></th>
  </tr>
+   </tr>
```

Comment:
I think this closing tag is unbalanced? Should be fine to just delete this line.

Reply:
will do!